.gitignore

```diff
@@ -223,6 +223,7 @@ node_modules/
 /ajax/libs/slim.js/2.[0-5].*/
 /ajax/libs/europa/4.0.0-alpha*/
 /ajax/libs/inferno-compat/*-alpha.*/
+/ajax/libs/fpo/[3-5].*
```

Comment:
@getify I think you can simply move this line to somewhere else like line 200 and it'll fix the conflict, don't need to give up :)